docs/section10/10-14.md

```diff
@@ -0,0 +1,177 @@
+                        - Set the modeled efficiency value: `modeled_efficiency_b = heating_system_b.efficiency_metric_values[x]`  
+                    - Else, do nothing and move on to the next metric: `Else:`   
+                - Check that modeled_efficiency_b does not equal Null: `if modeled_efficiency_b != Null:`     
+                    - Set expected_baseline_eff_b equal to 3.4 COPnf47: `expected_baseline_eff_b = 3.4`
+                    - Check if the modeled efficiency meets the expected baseline efficiency: `efficiency_matches_acceptable_value = expected_baseline_eff_b == modeled_efficiency_b`       
+                - Check if efficiency_matches_acceptable_value equals True, if it does then set equal to FALSE and perform logic below, if it equals False then go to Rule Assertion: `if efficiency_matches_acceptable_value == TRUE: `
+                    - Reset efficiency_matches_acceptable_value to False: `efficiency_matches_acceptable_value = FALSE`
+                    - Reset modeled_efficiency_b to equal Null: `modeled_efficiency_b = Null`  
+                    - Loop through each of the efficiency_metric_types associated with the heating system until HEAT_PUMP_EFFICIENCY_LOW_TEMPERATURE_NO_FAN is found. Get the associated value of the efficiency from the list of efficiency_metric_values associated with the heating_system_b : `For x in range(len(heating_system_b.efficiency_metric_types)):`
+                        - Check if the efficiency metric equals HEAT_PUMP_EFFICIENCY_LOW_TEMPERATURE_NO_FAN: `if heating_system_b.efficiency_metric_types[x] == "HEAT_PUMP_EFFICIENCY_LOW_TEMPERATURE_NO_FAN":` 
+                            - Set the modeled efficiency value: `modeled_efficiency_b = heating_system_b.efficiency_metric_values[x]`  
+                        - Else, do nothing and move on to the next metric: `Else:`   
+                    - Check that modeled_efficiency_b does not equal Null: `if modeled_efficiency_b != Null:`
+                        - Set expected_baseline_eff_b equal to 2.1 COPnf17: `expected_baseline_eff_b = 2.1`
+                        - Check if the modeled efficiency meets the expected baseline efficiency: `efficiency_matches_acceptable_value = expected_baseline_eff_b == modeled_efficiency_b`                         
+        **Rule Assertion:**  
+        - Case 1: if the modeled efficiency matches the expected baseline efficiency based on the size category or the modeled efficiency matches the most conservative efficiency across its size category and below and the efficiency was defined in the correct units then pass or capacity was not defined but the modeled efficiency meets the highest (most conservative) efficiency in the appropriate table based on system type then pass: `if efficiency_matches_acceptable_value == TRUE: PASS`    
```

Comment:
Are we okay with the approach where if we do not have all of the needed information we let it pass if it modeled the most stringent efficiency? I think we should when the aggregation factor is not defined because I think this will rarely be defined in simulation tools.

Reply:
I think in this case the logic should allow any of the efficiency values from system capacity ranges equal to or less than the defined capacity. The aggregated units don't necessarily need to be within the smallest capacity range.

Reply:
Smallest is most conservative that is why we selected it. Otherwise we could be passing for a more lenient scenario.

Reply:
Disregard my comment I agree with the logic that if aggregation factor is not specified then the efficiency needs to match the highest efficiency available in the data tables for that system type.